test/application_system_test_case.rb

```diff
@@ -1,5 +0,0 @@
-require "test_helper"
```

Comment:
Don't remove the files generated in initial commit. Also restore all of the `.keep` files you removed in this PR.